inc/auth.class.php

```diff
@@ -830,7 +830,7 @@ function login($login_name, $login_password, $noauto = false, $remember_me = fal
          if (!empty($this->user->fields['personal_token'])) {
```

Comment:
This test is already done in `getAuthToken()` method, maybe lines 830 to 834 could be replaced by only `$token = $this->user->getAuthToken();`.

Reply:
I agree, and I think we also can drop line 829